src/metrics/mod.rs

```diff
@@ -52,11 +42,17 @@ pub async fn init_prometheus<DB>(
 where
     DB: DBStatistics + Send + Sync + 'static,
 {
-    let registry = prometheus::default_registry();
+    // Add the process collector to the registry
+    if let Err(err) = kubert_prometheus_process::register(
+        DEFAULT_REGISTRY.write().sub_registry_with_prefix("process"),
```

Comment:
Perhaps a better idea is to expose `DEFAULT_REGISTRY` through a helper method. That way if we ever want to move/refactor the way it's used - we could keep the method, but change the underlying implementation/variable.

Reply:
Done